llvm.proj

```diff
@@ -15,7 +15,7 @@
 
     :: Try to locate installed VisualStudio VC environment.
     if "%VCINSTALLDIR%" == "" if exist "%VSWHERE_TOOLS_BIN%" (
-        for /f "tokens=*" %%a in ('"%VSWHERE_TOOLS_BIN%" -latest -prerelease -property installationPath') do (
+        for /f "tokens=*" %%a in ('"%VSWHERE_TOOLS_BIN%" -latest -property installationPath') do (
```

Comment:
I'd still leave prerelease, otherwise local builds on dogfood VS will.

Reply:
```suggestion
        for /f "tokens=*" %%a in ('"%VSWHERE_TOOLS_BIN%" -latest -prerelease -property installationPath') do (
```

Reply:
ok, makes sense. I guess we assume that the stable build image won't ever have a preview VS installed side-by-side.